modules/document_repository/README.md

```diff
@@ -0,0 +1,44 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents relevant to the study across multiple candidates.
+ Files may be uploaded and organized under any user-defined category.
+
+## Intended Users
+
+ Loris users can view, download and upload files, edit information about files, and delete files given appropriate permissions.
```

Comment:
Would write LORIS all upper case instead of Loris.
Also, instead of "download and upload files" would tend to write "download/upload files"